src/interpreter/interpreter.ts

```diff
@@ -166,6 +140,45 @@ export function interprete(interpreter: AbstractInterpreter, line: string, frame
   }
 }
 
+function interpreteExpression(expression: REPLExpression, interpreter: AbstractInterpreter, frame: Frame | undefined): ExecutionResult {
+  const error = [expression, ...expression.descendants].flatMap(_ => _.problems ?? []).find(_ => _.level === 'error')
+  if (error) throw error
+
+  if (expression.is(Sentence) || expression.is(Singleton) || expression.is(Class) || expression.is(Mixin)) {
```

Comment:
Podemos evitar este if si subimos el `  if (expression.is(Import)) {`?

Reply:
groso! quedó bien cheto subiendo el Import